schedule/jeos/sle/kvm/jeos-extratest.yaml

```diff
@@ -11,14 +11,15 @@ conditional_schedule:
 schedule:
     - jeos/grub2
     - jeos/firstrun
+    - console/consoletest_setup
+    - console/console_reboot
```

Comment:
Just curious. We do the reboot here but not for xen-hvm and others, whats the reason for it?

Reply:
I have just forgotten to update other files. Nevertheless, the apparent problem appears only in test jobs using virtio console.

Reply:
thx for explanation!